test/regridder/Regridder_Regression.ipynb

```diff
@@ -0,0 +1,235 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ab819831",
+   "metadata": {},
+   "source": [
+    "# Harmony Regridding Service (HRS) [regridder] regression tests\n",
+    "\n",
+    "This Jupyter notebook runs a suite of regression tests against some requests against the Harmony Regridding Service.\n",
+    "\n",
+    "These tests use the a MERRA-2 hourly collection of time averaged data ([M2T1NXSLV](https://cmr.earthdata.nasa.gov/search/concepts/C1276812863-GES_DISC.html)) and the ICESat-2/ATL16 Weekly Gridded Atmosphere collection ([ATL16](https://cmr.earthdata.nasa.gov/search/concepts/C2153575232-NSIDC_CPRD.html)) available from GES DISC and NSIDC, respectively.\n",
```

Comment:
I updated this line, which previously referred to GPM_3IMERGHH. But the tests use M2T1NXSLV and ICESat-2/ATL16.